drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/pages/FactPatternConstraintsPageViewImpl.ui.xml

```diff
@@ -41,8 +44,8 @@
           <gwt:ScrollPanel ui:field="availableFieldsContainer" styleName="{res.css.wizardDTableList}" height="220px" width="180px"/>
         </gwt:VerticalPanel>
         <gwt:VerticalPanel ui:field="buttonBar" styleName="{res.css.wizardDTableButtons}">
-          <b:Button ui:field="btnAdd" enabled="false">&gt;&gt;</b:Button>
-          <b:Button ui:field="btnRemove" enabled="false">&lt;&lt;</b:Button>
+          <b:Button ui:field="btnAdd" enabled="false" icon="ANGLE_RIGHT" width="50px"/>
```

Comment:
Instead of specifying width, you could use iconFixedWidth=true, which adds the default width for icons.